pkg/ui/src/Header/Header.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+	import Logo from './Logo.svelte';
+	export let title = 'Stock';
+	const links = [
+		{
+			label: 'Stock',
+			href: '/inventory/stock'
+		},
+		{
+			label: 'Inbound',
+			href: '/inventory/inbound'
+		},
+		{
+			label: 'Outbound',
+			href: '/inventory/outbound'
+		}
+	];
+	export let currentUrl = 'stock';
```

Comment:
1. I can see how this could go wrong:
- passing a current "url", which isn't a proper url string, from the outside
- matching a passed "url" to a label (which might get translated at some point)

2. "stock" isn't a proper url string, what you want to use is `$page.url.pathname` and match it against each `href` to get the current location (the same way you would use react router context)

Reply:
can I import page outside of a sveltekit page?